packages/components/pie-link/src/index.ts

```diff
@@ -120,14 +121,14 @@ export class PieLink extends LitElement implements LinkProps {
             <a
                 data-test-id="pie-link"
                 class="c-link"
-                tag=${this.tag}
-                variant=${this.variant}
-                size=${this.size}
-                underline=${this.underline}
+                tag=${this.tag || 'a'}
+                variant=${this.variant || 'default'}
+                size=${this.size || 'medium'}
+                underline=${this.underline || 'default'}
                 ?isBold=${this.isBold}
                 ?isStandalone=${this.isStandalone}
                 ?hasVisited=${this.hasVisited}
-                href=${this.href || nothing}
+                href=${ifDefined(this.href)}
```

Comment:
Just checking. Will this line remove the href attribute if it is not defined? Should it default to empty instead if it does remove it?